src/modules/access-wallet/software/handlers/helpers.js

```diff
@@ -0,0 +1,15 @@
+import { isHexString } from 'ethereumjs-util';
+
+const SOFTWARE_WALLET_TYPES = {
+  KEYSTORE: 'keystore',
+  MNEMONIC: 'mnemonic',
+  PRIVATE_KEY: 'private-key',
+  OVERVIEW: 'overview'
+};
```

Comment:
lets import these from `configWalletTypes`

Reply:
These are actually different they are used in the route query definitions. overview does not exists in the configWalletTypes.